float64[] id_list
float64[] x_list
float64[] y_list
